templates/KnockoutSpa/ClientApp/boot.ts

```diff
@@ -1,8 +1,8 @@
 import './css/site.css';
 import * as ko from 'knockout';
-import { createHistory } from 'history';
 import './webpack-component-loader';
 import AppRootComponent from './components/app-root/app-root';
+var createHistory = require('history').createBrowserHistory
```

Comment:
So it needs to be updated because when the previous `import {}` way was done, with the new History, it doesn't call the correct methods and ends up erroring that "createBrowserHistory" doesn't exist. I've tried the other ways suggested on the packages readme (https://github.com/mjackson/history) but only this one works). Updating it to `const` now.